static/js/util/api.js

```diff
@@ -79,3 +79,7 @@ function fetchJSONWithCSRF(input, init) {
 export function getCourseList() {
   return fetchJSONWithCSRF('/api/v0/courses/');
 }
+
+export function getUserImage() {
+  return fetchJSONWithCSRF('/api/v0/profile/'+SETTINGS.username);
```

Comment:
It would be cleaner to have `getUserImage` take an argument `username` and use that instead of `SETTINGS.username`, in case we wanted to use the function for another user in the future. Instead, have the code which dispatches the `fetchUserImage` action take `SETTINGS.username` as an argument